stdlib/3/builtins.pyi

```diff
@@ -38,9 +38,26 @@ class object:
     def __setattr__(self, name: str, value: Any) -> None: ...
     def __eq__(self, o: object) -> bool: ...
     def __ne__(self, o: object) -> bool: ...
+    def __lt__(self, o: object) -> bool: ...
+    def __le__(self, o: object) -> bool: ...
+    def __gt__(self, o: object) -> bool: ...
+    def __ge__(self, o: object) -> bool: ...
     def __str__(self) -> str: ...
     def __repr__(self) -> str: ...
     def __hash__(self) -> int: ...
+    def __del__(self) -> None: ...
```

Comment:
These don't seem to exist in Python 3.5.1: `__del__`, `__bytes__`, `__bool__`, `__getattr__`, `__get__`, `__set__`.